src/app/clusters/client-monitoring-server/client-monitoring-server.h

```diff
@@ -18,6 +18,25 @@
 #pragma once
 
 #include <app-common/zap-generated/cluster-objects.h>
+#include <app/CommandHandler.h>
 #include <app/ConcreteCommandPath.h>
 #include <app/util/af-types.h>
 #include <app/util/basic-types.h>
+
+class ClientMonitoringServer
+{
+public:
+    ClientMonitoringServer(){};
```

Comment:
Where is the state keeping re: how to register the delegates so that commands lead to side-effects? Will it be a subsequent PR?

Reply:
The side-effect behaviours will come in a subsequent PR.

This is the ticket for that work. The design isn't decided yet.
#24258